src/js/components/FilterButtons.js

```diff
@@ -36,8 +36,8 @@ class FilterButtons extends React.Component {
     let filterCount = this.getCount(itemList);
 
     return filters.map((filter) => {
-      let classSet = classNames({
-        'button button-stroke button-inverse': true,
+      let classSet = classNames('button button-stroke',{
```

Comment:
This is a good idea.
I suggest to add this only when needed. 
Could we make usage of this generalizing thing already?